ocp_install_env.sh

```diff
@@ -49,7 +49,7 @@ function extract_installer() {
 function clone_installer() {
   # Clone repo, if not already present
   if [[ ! -d $OPENSHIFT_INSTALL_PATH ]]; then
-    sync_repo_and_patch go/src/github.com/openshift/installer https://github.com/openshift/installer.git
+    sync_repo_and_patch go/src/github.com/openshift/installer https://github.com/openshift/installer.git https://patch-diff.githubusercontent.com/raw/openshift/installer/pull/2657.patch https://patch-diff.githubusercontent.com/raw/openshift/installer/pull/2666.patch https://patch-diff.githubusercontent.com/raw/openshift/installer/pull/2673.patch
```

Comment:
I wonder if we should break out just the installer fixes so that folks don't need to apply the workarounds manually?

Reply:
Where would we break them out to?

Reply:
I just meant decouple them from the ipv6 testing, which you've now done :+1: